app/Livewire/Project/Database/BackupEdit.php

```
@@ -98,58 +161,173 @@ public function syncData(bool $toModel = false)
+                    $this->backup->disable_local_backup = $this->saveS3 && $this->disableLocalBackup;
+
+                    $this->backup->pgbackrest_backup_type = $this->pgbackrestBackupType;
+                    $this->backup->pgbackrest_compress_type = $this->pgbackrestCompressType;
+                    $this->backup->pgbackrest_compress_level = $this->pgbackrestCompressLevel;
+                    $this->backup->pgbackrest_log_level = $this->pgbackrestLogLevel;
+                    $this->backup->pgbackrest_archive_mode = $this->pgbackrestArchiveMode;
+
+                    $this->customValidate();
+                    $this->backup->save();
+                    $this->syncPgbackrestRepos();
+                });
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'syncData'. (undefined)

(StaticAccess)